src/utils/helpers/list/useGetFilterDefinitions.ts

```diff
@@ -1,21 +1,23 @@
 import { DocumentNode } from 'graphql'
 import { DateTime, DurationObjectUnits } from 'luxon'
-import { today } from '../dateAndTime/parseDateRange'
+import { today } from '../../dateAndTime/parseDateRange'
 import {
   ApplicationOutcome,
   ApplicationStatus,
   AssignerAction,
   ReviewerAction,
-} from '../generated/graphql'
-import getApplicantFilterList from '../graphql/queries/applicationListFilters/getApplicantFilterList'
-import getAssignerFilterList from '../graphql/queries/applicationListFilters/getAssignerFilterList'
-import getOrganisationFilterList from '../graphql/queries/applicationListFilters/getOrganisationFilterList'
-import getReviewersFilterList from '../graphql/queries/applicationListFilters/getReviewersFilterList'
-import getStageFilterList from '../graphql/queries/applicationListFilters/getStageFilterList'
-import { FilterDefinitions, GetFilterListQuery, NamedDates } from '../types'
-import { useLanguageProvider } from '../../contexts/Localisation'
+} from '../../generated/graphql'
+import getApplicantFilterList from '../../graphql/queries/applicationListFilters/getApplicantFilterList'
+import getAssignerFilterList from '../../graphql/queries/applicationListFilters/getAssignerFilterList'
+import getOrganisationFilterList from '../../graphql/queries/applicationListFilters/getOrganisationFilterList'
+import getReviewersFilterList from '../../graphql/queries/applicationListFilters/getReviewersFilterList'
+import getStageFilterList from '../../graphql/queries/applicationListFilters/getStageFilterList'
+import { FilterDefinitions, GetFilterListQuery, NamedDates } from '../../types'
+import { useLanguageProvider } from '../../../contexts/Localisation'
+import { USER_ROLES } from '../../data'
+import LIST_FILTERS from '../../data/listFilters'
 
-export const useApplicationFilters = () => {
+export const useGetFilterDefinitions = (defaultFilters: string[]) => {
```

Comment:
As suggested above, I think it would be better to read the prefs inside here rather than pass in a parameter.

Reply:
Yes, it could be better... But there is a thing with getting the userRole.

Reply:
> But there is a thing with getting the userRole

What thing? 😕